docs/how-it-works.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_position: 2
+---
+
+# How It Works
+
+Learn how Divvi Up works and how it protects privacy.
+
+## Breakdown
+
+Divvi Up ingests measurements and aggregates them, while protecting the privacy
+of each individual measurement. This can be broadly broken down into several
+steps, described in the following sections.
+
+<!-- TODO(inahga): Consider making this a more detailed `mermaid` diagram -->
+
+![](../static/how-it-works.png#gh-dark-mode-only)
+![](../static/how-it-works-light.jpg#gh-light-mode-only)
+
+See the [Distributed Aggregation Protocol (DAP)][DAP] for a highly detailed
+specification of the protocol that underpins Divvi Up.
+
+### Step 0: Subscriber sets up Divvi Up
+
+Subscribers are users of Divvi Up, such as app developers, who wish to collect
+quantitative measurements about their end-user applications.
+
+The subscriber chooses a DAP client library, which is used to interact with any
+DAP-compatible measurements aggregator, such as the aggregators provided by
+Divvi Up and its partners. Client libraries for [Rust
+(janus_client)][janus_client] and [TypeScript (divviup-ts)][divviup-ts] are
+available.
+
+For each measurement the subscriber wishes to aggregate, the subscriber creates
+a task. A task encodes metadata about how the measurement will be made, which
+aggregators will be used, and which aggregation function to use. The task's
+parameters are shared with both aggregators.
+
+### Step 1: Client generates measurements
+
+The subscriber writes and deploys app functionality that takes measurements
+corresponding to each task. They invoke a DAP client library and provide some of
+the task parameters to it, such as the task identifier, measurement type, and
+the aggregator servers being used.
+
+For each measurement, the subsequent steps are performed.
+
+### Step 2: Client shards the measurement
+
+The app uses the DAP client library to randomly shard the measurement into two
+parts, known as report shares. One report share is for the leader aggregator,
+and the other for the helper aggregator.
+
+Each aggregator advertises a public key corresponding to its private key. The
+client library encrypts each share with the respective aggregator's public key.
+
+### Step 3: Client sends shares to the aggregators
+
+The DAP client library sends each report share via HTTPS to the leader and
+helper aggregators.
```

Comment:
This isn't quite correct, because the client doesn't transmit a report share directly to the helper. I see there's a note about this below, but why not just say this plainly in this paragraph?

Reply:
I went with removing the admonitions and plainly stating reality. It reads well enough IMO, but we should have a more precise diagram to accompany this doc. I can work on this in a future PR.